templates/devtest/gitea-ui.tmpl

```diff
@@ -177,12 +177,13 @@
 
 		<h2>Dropdown with SVG</h2>
 		<div>
-			<div class="ui dropdown" style="border: 1px red dashed" data-tooltip-content="border for demo purpose only">
-				<span class="text">simple</span>
+			<div class="ui dropdown tw-border tw-border-red tw-border-dashed" data-tooltip-content="border for demo purpose only">
+				<span class="text">search-input & flex-item in menu</span>
```

Comment:
```suggestion
				<span class="text">search-input &amp; flex-item in menu</span>
```

Reply:
Well, I think using a single `&` (with non-word following it) is not bad in modern HTML, because it could always parsed correctly.

If you strongly prefer to use `&amp;`, feel free to commit this change.

Reply:
I only really recommend it because standard-html escaping also escapes it:

https://github.com/sindresorhus/escape-goat/blob/main/index.js

But I see:

> an ampersand which is not followed by an alphanumeric character or a hash does not need to be escape

Reply:
Thought I guess I do feel strong about it because this line confused me while visually parsing this line. Also, WHATWG [recommends](https://html.spec.whatwg.org/#syntax) escaping it.

<img width="1150" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/2b969a5d-e1d2-4585-891b-0946445a5d40">